apps/api/v2/test/fixtures/repository/team.repository.fixture.ts

```diff
@@ -21,7 +21,7 @@ export class TeamRepositoryFixture {
   }
 
   async delete(teamId: Team["id"]) {
-    return this.prismaWriteClient.team.delete({ where: { id: teamId } });
+    return this.prismaWriteClient.team.deleteMany({ where: { id: teamId } });
```

Comment:
deleteMany ? is the return type more adequate ?

Reply:
I put deleteMany because it does not error if team does not exist

Reply:
actually lemme put it back as it was